exercises/practice/lens-person/.meta/proof.ci.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars */
+import { Person } from '../person';
+import { Name } from '../name';
+import { Born } from '../born';
+import { Address } from '../address';
+import { Lens } from '../lens';
```

Comment:
Nit: can you order this so we don't have discussions later down the line/people using automatic ordering in vscode.